lib/App/DuckPAN.pm

```diff
@@ -426,6 +429,20 @@ sub phrase_to_camel {
 	return $camel;
 }
 
+# Normalize an Instant Answer name to a standard form.
+# Returns undef if an IA matching the given name cannot be found.
+sub normalize_ia_name {
+	my ($self, $name) = @_;
+	my $ia = $name =~ /_/
+		? DDG::Meta::Data->get_ia(id => $name)
+		: DDG::Meta::Data->get_ia(id => $self->camel_to_underscore($name))
+		or $self->emit_and_exit(1, "No Instant Answer found with name '$name'");
+	my $mod_name = $ia->{perl_module};
+	$mod_name =~ /^DDG::(?:Goodie|Spice)(?:::[^:]+)*::([^:]+)$/
```

Comment:
Uhh... *shrug*

I think this is so we get the CamelCase name? It might not be required.